tooling/config-eslint/utils/overrides.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+    overrides: [
+        {
+            files: ['*.cjs', '*.js', '*.mjs'],
+            parser: '@typescript-eslint/parser',
+        },
+        {
+            extends: ['plugin:@typescript-eslint/disable-type-checked'],
+            files: ['**/.eslintrc.*'],
```

Comment:
No TSConfig has `.eslintrc.*` files included, so to get them type-checked, we'd have to use [`allowDefaultProject`](https://typescript-eslint.io/packages/parser#allowdefaultproject) for them:

```plaintext
  0:0  error  Parsing error: /Users/josh/repos/typehero/.eslintrc.js was not found by the project service. Consider either including it in the tsconfig.json or including it in allowDefaultProject
```

```plaintext
/home/runner/work/typehero/typehero/apps/auth-proxy/.eslintrc.cjs
  0:0  error  Parsing error: /home/runner/work/typehero/typehero/apps/auth-proxy/.eslintrc.cjs was not found by the project service. Consider either including it in the tsconfig.json or including it in allowDefaultProject
```

It's easier to just ... not type-check them for now.